### DISCUSSION TITLE
How to make /.singularity.d/libs/libfakeroot.so visible to fakeroot also visible to normal user?

### DISCUSSION THREAD

**Discussion by Heng-Zhou** [sylabs/singularity#3381] [Q&A]
I installed package _libfakeroot_ using command `apt install -y libfakeroot` in the definition file in hope that `/.singularity.d/libs/libfakeroot.so` could be seen by normal user. But because the user is (fake) root when the package _libfakeroot_  is installed, this file is only visible when I shell into the container as a fake root user (`shell --fakeroot`). If I shell into the container as a normal user (without `--fakeroot`), the directory `/.singularity.d/libs/` is empty. Is there any way to make `/.singularity.d/libs/libfakeroot.so` visible to fakeroot also visible to a normal user? Thank you.

**Comment by dtrudg** [comment]
Bit confused here - SingularityCE has no handling of libfakeroot. If you install it in the container then it won't be bound into `/.singularity.d/libs` as that's where SingularityCE puts GPU libs etc. that are bound in from the host.

Are you sure you aren't using Apptainer... but running it via the `singularity` alias it installs? Apptainer does use libfakeroot, but SingularityCE does not. What do you get from `singularity --version`?
